app/views/movies/index.html.erb

```diff
@@ -0,0 +1,179 @@
+<div class="row mb-3">
+  <div class="col-md-12">
+    <h1>
+      List of all movies
+    </h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="row mb-3">
+  <div class="col-md-8 offset-md-2">
+    <h2>
+      Add a new movie
+    </h2>
+
+    <form action="/insert_movie" method="post">
+      <div class="form-group">
+        <label for="title">
```

Comment:
This quarter I landed on the convention of `_box` for `label[for]` and `input|textarea|select[id]` names, to evoke the visual/UI nature of the purpose of it. E.g.,

```html
<label for="body_box">Body</label>
<textarea id="body_box></textarea>
```

Thoughts?

Reply:
I like that convention. I think more versatile but it's just as descriptive as having something like

```html
<label for="body_input">Body</label>
<textarea id="body_input></textarea>
```